Source/CarthageKit/Git.swift

```diff
@@ -153,6 +153,7 @@ public func contentsOfFileInRepository(_ repositoryFileURL: URL, _ path: String,
 public func checkoutRepositoryToDirectory(
 	_ repositoryFileURL: URL,
 	_ workingDirectoryURL: URL,
+	force: Bool,
```

Comment:
Why was this added/needed?

Reply:
The original `checkoutRepositoryToDirectory` function was a forced checkout, and in `checkoutSubmodule` there was a non-forced checkout. So to reuse the `checkoutRepositoryToDirectory` function, I added the force parameter.